core/renderers/common/debug.js

```diff
@@ -43,6 +38,8 @@ exports.isDebuggerEnabled = isDebuggerEnabled;
  * Turn on the blocks debugger.
  * @package
  * @alias Blockly.blockRendering.debug.startDebugger
+ * @deprecated September 2022. Use the rendering debugger in @blockly/dev-tools.
```

Comment:
Checking a few other place where dates appear in `@deprecated` annotations, it appears that they are usually the date of the release in which the `@deprecated` tag was added, so shouldn't this be "March 2022"?

Reply:
Yup, good call. Fixed.